README.md

```diff
@@ -43,4 +43,95 @@ If you'd like to try out the plugin pre-release, you can build it from source an
+```
+
+Open the `hello-spring-boot` example from `google-container-tools/examples` directory with your IntelliJ IDE. You can either point to the directory or to the Maven build file (`pom.xml`). The project opens and loads:
+
+<img src="docs/images/sb-hello-world-project.png" alt="opened Spring Boot hello world project" width="500"/> 
+
+This project is a very simple web application created with [the popular Spring Boot framework](https://spring.io/projects/spring-boot). It uses the [Jib Maven plugin](https://github.com/GoogleContainerTools/jib) to build a container image for the project, without needing to create a Dockerfile.
+
+Once the project loads, the plugin will detect the Skaffold configuration and prompt to create the Kubernetes targets. The notification shows:
+
+<img src="docs/images/k8s-skaffold-notification.png" alt="Kubernetes with Skaffold notification" width="400"/> 
+
+Click `Create run configurations for Kubernetes with Skaffold` link to automatically create Kubernetes deployment and continuous development IDE run targets for the project:
+
+<img src="docs/images/k8s-skaffold-run-targets.png" alt="Kubernetes with Skaffold pre-configured run targets" width="400"/> 
+
+Now the new run targets can be used to build the project and deploy it to Kubernetes or develop on Kubernetes cluster continuously. With continuous development, the plugin uses Skaffold to watch the sources and dependencies of your project for changes, so that on any change, Skaffold builds and deploys your application to a Kubernetes cluster.
+
+### Specifying image repository
+
+However, before we can deploy and develop, we need to make sure we have access to the image repository where the project image is about to be pushed. By default the project is configured to use [Google Container Registry](https://cloud.google.com/container-registry/) and a development project for the plugin which you probably don’t have access to. Once you have your repository set up ([Google Container Registry](https://cloud.google.com/container-registry/), [DockerHub](https://hub.docker.com/), private repository, etc.), you can edit the run targets and specify it as a *default image repository* in run target settings:
+
+![specify your repository in run target settings](docs/images/default-image-repo-settings.png)
+
+*Note*: this step is not required when you work with your own Kubernetes manifests and Skaffold configuration where you specify a repository and an image name that are accessible to you.
+
+### Continuous development on Kubernetes
+
+Now it is time to deploy the project to Kubernetes or develop on a Kubernetes cluster from your IDE! Before we start, make sure [all required dependencies](https://github.com/GoogleContainerTools/google-container-tools-intellij#prerequisites-and-required-dependencies) are available on your machine. Click the run action for `Develop on Kubernetes` to start development cycle on your Kubernetes cluster:
```

Comment:
> Now it is time to deploy the project to Kubernetes or develop on a Kubernetes cluster from your IDE!

Now that we have sections, this sentence feels off as the first sentence of the "continous..." section. What about changing it to something like:

> Now you can set up a continuous development iteration cycle in your IDE. With this setup, [move the sentence you added explaining what continuous development does here].

Also wrt to "Before we start, make sure all"; should we move this to the top "Getting started" section? I feel like this is the first thing they should ensure.

Reply:
Agree, section heading pretty much replaces that sentence, good thinking! Moved required dependencies to getting started as well, makes sense.